src/test/files/maven/devkit-latest-version/devkit-is-latest-version/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.mule.tools.devkit.sonar</groupId>
+    <artifactId>certification-plugin</artifactId>
+    <packaging>sonar-plugin</packaging>
+    <version>SOME-SNAPSHOT</version>
+    <parent>
+	<groupId>org.mule.tools.devkit</groupId>
+	<artifactId>certified-mule-connector-parent</artifactId>
+	<version>3.9.0</version>
+    </parent>
+    <dependencies>
```

Comment:
Also, you should have another test case with an older version of DevKit, to see if the rule non-compliance is reported.